packages/react/src/components/ContentItemHorizontal/ContentItemHorizontal.js

```diff
@@ -91,13 +91,22 @@ ContentItemHorizontal.propTypes = {
    * | `copy` | YES      | String    | Link text.                                 |
    * | `href` | YES      | String    | URI for internal or external resource.     |
    */
-  cta: PropTypes.arrayOf(
-    PropTypes.shape({
-      type: PropTypes.oneOf(['local', 'external']).isRequired,
-      href: PropTypes.string.isRequired,
-      copy: PropTypes.string.isRequired,
-    })
-  ).isRequired,
+  cta: PropTypes.shape({
+    heading: PropTypes.string,
+    items: PropTypes.arrayOf(
+      PropTypes.shape({
+        style: PropTypes.oneOf(['text', 'card', 'button', 'feature']),
```

Comment:
Thanks @asudoh. I think at the moment the CTA in this component only allows for `text` style. [Func spec](https://ibm.ent.box.com/file/663532447262)
```suggestion
        style: PropTypes.oneOf(['text']),
```